holidays/countries/jersey.py

```diff
@@ -0,0 +1,240 @@
+        kwargs.setdefault(
+            "rule", SUN_TO_NEXT_MON if dt < date(2004, OCT, 12) else self._observed_rule
+        )
+        return super()._add_observed(dt, **kwargs)
+
+    def _populate_public_holidays(self) -> None:
+        # Earliest available piece of law available is from 1952.
+        if self._year <= 1951:
+            return None
+
+        # New Year's Day.
+        # Available online source shown that this was celebrated since at least 1952.
+        # Was briefly removed in 1983 only to be added back again before that came to effect.
+
+        # New Year's Day
+        self._add_observed(self._add_new_years_day("New Year's Day"))
+
+        # Good Friday
+        self._add_good_friday("Good Friday")
+
+        # Easter Monday
+        self._add_easter_monday("Easter Monday")
+
+        # May Day
+        # This only starts in 1980 (instead of 1978) for Jersey.
+        # The date is not moved in 2020 (unlike in the UK) as there's already VE Day Celebrations.
+
+        # May Day bank holiday (first Monday in May)
+        if self._year >= 1980:
+            name = "May Day"
+            if self._year == 1995:
+                self._add_holiday_may_8(name)
+            else:
+                self._add_holiday_1st_mon_of_may(name)
+
+        # Spring bank holiday
+        # Current Pattern started in 1970 for Jersey.
+
+        # Spring bank holiday (last Monday in May)
+        if self._year >= 1970:
+            spring_bank_dates = {
+                2002: (JUN, 4),
+                2012: (JUN, 4),
+                2022: (JUN, 2),
+            }
+            name = "Spring Bank Holiday"
+            if self._year in spring_bank_dates:
+                self._add_holiday(name, spring_bank_dates[self._year])
+            else:
+                self._add_holiday_last_mon_of_may(name)
+
+        # Whit Monday.
+        # Was in-use prior to Spring bank holiday adoption.
+
+        if self._year <= 1969:
+            # Whit Monday.
+            self._add_whit_monday("Whit Monday")
+
+        # Late Summer Bank Holiday
+        # Current Pattern started in 1970. Was previously first Monday of September for Jersey.
+
+        # Late Summer bank holiday (last Monday in August)
+        summer_bank_holiday = "Late Summer Bank Holiday"
+        if self._year >= 1970:
+            self._add_holiday_last_mon_of_aug(summer_bank_holiday)
+        else:
+            self._add_holiday_1st_mon_of_sep(summer_bank_holiday)
+
+        # Christmas Day
+        christmas_day = self._add_christmas_day("Christmas Day")
+
+        # Boxing Day
+        boxing_day = self._add_christmas_day_two("Boxing Day")
+
+        if self._year >= 2004:
+            self._add_observed(christmas_day, rule=SAT_SUN_TO_NEXT_MON_TUE)
+            self._add_observed(boxing_day, rule=SAT_SUN_TO_NEXT_MON_TUE)
+        else:
+            self._add_observed(christmas_day, rule=SUN_TO_NEXT_TUE)
+            self._add_observed(boxing_day)
```

Comment:
```suggestion
        self._add_observed(christmas_day)
        self._add_observed(boxing_day)
```

Reply:
This seems quite handy - gonna remember it for future PRs 👀